components/class_loader/debug_class_loader.rst

```diff
@@ -1,9 +0,0 @@
-.. index::
-    single: ClassLoader; DebugClassLoader
-    
-Debugging a Class Loader
```

Comment:
The url of this page should be added in the redirection map (the only reason why it was done this way rather than removing the page was that because symfony.com was not supporting different redirections per versions of the doc, so it was impossible to remove the page)